lib/livebook/runtime/erl_dist/runtime_server.ex

```diff
@@ -772,7 +772,9 @@ defmodule Livebook.Runtime.ErlDist.RuntimeServer do
 
   defp evaluator_tmp_dir(state) do
     if tmp_dir = state.tmp_dir do
-      Path.join(tmp_dir, "tmp")
+      path = Path.join(tmp_dir, "tmp")
+      File.mkdir_p!(path)
+      path
```

Comment:
@jonatanklosko do you prefer to enforce this here or in the evaluator before we write to the directory?

Reply:
Let's do it before writing the file :)

Reply:
@jonatanklosko  - is there another place you would like to do this? If so let me know where?

Reply:
Do it just before you call File.write! before calling :epp.

Reply:
I removed the mkdir_p from the runtime_server and added just before epp call.